extension/lib/time_tracker_view.js

```diff
@@ -61,15 +62,15 @@ class TimeTrackerView {
     Updates reading timer based on current sentence.
     */
     updateTimer(sentenceId) { // Call everytime the tracker moves.
-        // TODO: Should also update timer if a user is using the autoread mode. 
         let total_words = this.doc.getNumWordsFromSentenceTilEnd(sentenceId);
-        let currentSpeed = this.reading_speed;
         // if (this.auto_mode) { currentSpeed = this.reading_speed } else { currentSpeed = avg_read_speed };
-        // TODO: Update time_remaining to be in seconds than minutes.
-        // See https://github.com/benfrancis314/reading_project/issues/61
-        let time_remaining = total_words/currentSpeed; // in minutes
-        this.time_remaining = Math.ceil(time_remaining);
-        document.getElementById("timerNumber").innerHTML = this.time_remaining;
+        let time_remaining_m = total_words / this.reading_speed; // in minutes
+        this.time_remaining_ms = time_remaining_m * 60 * 1000;
+
+        let m = Math.floor(time_remaining_m);
+        let s = Math.floor((time_remaining_m - m) * 60);
```

Comment:
It doesn't matter but shouldn't this be ceiling?

Reply:
(Min floor, seconds ceiling)

Reply:
(Again, this doesn't matter, since it's a fraction of a second. But if there were 1.5 seconds, it'd be better to say they have 2 seconds left than 1, right?)

Reply:
I wanted to see the last paragraph having 0 second lol, but yeah it almost never happens.
Changed to math.round as a compromise ;o